src/runner/bootstrapper.ts

```diff
@@ -186,6 +188,24 @@ export default class Bootstrapper {
         RemoteBrowserProvider.canDetectLocalBrowsers = false;
     }
 
+    private static async _checkThatTestsCanRunWithoutDisplay (browserInfoSource: BrowserInfoSource[]): Promise<void> {
+        for (const browserInfo of browserInfoSource) {
+            if (browserInfo instanceof BrowserConnection)
```

Comment:
I still think that we can remove this condition. From the auth check function too, but it's better to do it in a separate PR.